realestatinator/__manifest__.py

```diff
@@ -11,6 +11,8 @@
 	],
 	'data': [
 		'security/ir.model.access.csv',
+		'views/estate_property_views.xml',
```

Comment:
you can fixup all the commit "Chapter 5" inside the same commit. Check git documentation on interactive rebase / fixup if needed :)